src/test/java/Hw17.java

```diff
@@ -0,0 +1,26 @@
+import org.testng.annotations.Test;
+
+public class Hw17 extends BaseTest {
+
+    @Test(description = " Adding a song to Abdi's Playlist ")
+    public static void addSongToPlaylist() throws InterruptedException  {
+
+        browserConfigs();
+        provideEmail("dcabdi@gmail.com");
+        providePassword("te$t$tudent");
+        clickSubmitBtn();
+        Thread.sleep(5000);
+        allSongsTab();
+        songChoice();
+        addSongToPlaylistBtn();
+        abdisPlaylist();
+        validation();
+        tearDownBrowser();
+
+        allSongsTab();
+        songChoice();
+        addSongToPlaylistBtn();
+        abdisPlaylist();
+        validation();
```

Comment:
Please delete all these methods because they are repetitive

For exampe

```
@Test(description = " Adding a song to Abdi's Playlist ")
    public static void addSongToPlaylist() throws InterruptedException  {

        browserConfigs();
        provideEmail("dcabdi@gmail.com");
        providePassword("te$t$tudent");
        clickSubmitBtn();
        Thread.sleep(5000);
        allSongsTab();
        songChoice();
        addSongToPlaylistBtn();
        abdisPlaylist();
        validation();
        tearDownBrowser();

       
    }
```